modules/cluster-logging-rn-5.2.11.adoc

```diff
@@ -0,0 +1,60 @@
+[id="cluster-logging-release-notes-5-2-11"]
+= Logging 5.2.11
+This release includes https://access.redhat.com/errata/RHBA-2022:5012[RHBA-2022:5012-OpenShift Logging Bug Fix Release 5.2.11]
+
+[id="openshift-logging-5-2-11-bug-fixes"]
+== Bug fixes
+* Before this update, clusters configured to perform CloudWatch forwarding wrote rejected log files to temporary storage, causing cluster instability over time. With this update, chunk backup for CloudWatch has been disabled, resolving the issue. (link:https://issues.redhat.com/browse/LOG-2635[LOG-2635])
+
```

Comment:
These two bugs are in 5.2.11 too.
https://issues.redhat.com/browse/LOG-2671
https://issues.redhat.com/browse/LOG-2670